quickstart-node/src/services/monday-code/environment-variables.service.ts

```diff
@@ -0,0 +1,18 @@
+import { EnvironmentVariablesManager } from '@mondaycom/apps-sdk';
+import { GetKeyOptions, JsonValue, MondayCodeKeyValueManager } from '@my-types/monday-code-sdk';
+
+export class EnvironmentVariablesService implements MondayCodeKeyValueManager {
+  // eslint-disable-next-line no-use-before-define
+  public static instance: EnvironmentVariablesService = new EnvironmentVariablesService();
```

Comment:
Maybe make the instance private and implement the `new` in `getInstance()`?

something like
```
  public static getInstance(): Singleton {
    if (!Singleton.instance) {
      Singleton.instance = new Singleton();
    }
    return Singleton.instance;
  }
```